tests-ng/handlers/configure_nvme.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from plugins.shell import ShellRunner
+from plugins.dpkg import Dpkg
+from plugins.module import Module
+import json
+import os
+
+# Define variables for the IP address, NVMe device, and subsystem name
+IP_ADDRESS="127.0.0.1"
+NVME_DEVICE="/tmp/nvme.img"
+SUBSYSTEM_NAME="testnqn"
+PORT_NUMBER="4420"
+TRTYPE="tcp"
+ADRFAM="ipv4"
+
+REQUIRED_NVME_MODULE = [
+            {"nvme_module": "nvme_tcp", "status": None},
+            {"nvme_module": "nvmet-tcp", "status": None},
```

Comment:
While (un)loading modules with `-` in the name works, `lsmod` or `/proc/modules` always use `_`. Please adapt this.